inputs/mvs_config.json

```
@@ -331,17 +287,17 @@
             "file_name": "pv_gen_merra2_2014_eff1_tilt40_az180.csv",
             "installedCap": {
                 "unit": "kWp",
-                "value": 0
-            },
-            "maximumCap": {
-                "unit": "kWp",
-                "value": 1000
+                "value": 1500
             },
             "label": "PV plant (mono)",
             "lifetime": {
                 "unit": "year",
                 "value": 30
             },
+            "maximumCap": {
+                "unit": " kWp",
+                "value": 1000
```

```suggestion
                "value": None
```

I guess it should be None. You could try whether this works!

---

I think I tried that and I am not sure anymore if it worked, I will try tomorrow

---

I does not work :(